scipy/optimize/tests/test_minpack.py

```diff
@@ -321,7 +321,7 @@ def test_full_output(self):
                               args=(self.y_meas, self.x),
                               full_output=True)
         params_fit, cov_x, infodict, mesg, ier = full_output
-        assert_(ier in (1,2,3,4), 'solution not found: %s' % mesg)
+        assert_(ier in (1,2,3,4), f'solution not found: {mesg}')
```

Comment:
It's unclear how I got myself in this mess of having to fix old-style string replacements in unrelated files... some sort of chain reaction from a seeminly minor change, and now I'm not sure where it all started. Sorry for the extra review : /